README.md

```diff
@@ -1,47 +1,94 @@
+ program and it was based on an observation model developed there.
+A simplified UML model can be found
+ [here](https://raw.github.com/VarioML/VarioML/master/xml/lsdb_main/uml/varioml.jpg).
 
+__VarioML is a collaborative, community-driven specification in active
+ development.__
+If you'd like to collaborate with us, your first step should be to contact
+ *admin* at *varioml.org*, as documentation is not yet complete.
 
-__VarioML is a collaborative, community-driven specification in active development.__
-Your first step should be to contact *admin* at *varioml.org*, as documentation is not yet complete.
 
-__Important__:
-- All new implementations should use validation to ensure consistency with the core specification. Contact *admin* at *varioml.org* for help with generating validation scripts using Schematron.
-- Use standard ontologies for the content, such as the [Sequence Ontology](http://www.sequenceontology.org/) and
- [VariO](http://variationontology.org/). Some LSDB specific terms are defined using [SKOS vocabularies](https://github.com/VarioML/VarioML/tree/master/ontology/skos)
-- Source attributes in database xrefs (such as *gene* and *ref_seq*) as well as ontology terms should use database abbreviations defined in the [MIRIAM registry](http://identifiers.org). For example, use *hgnc.symbol* for gene names, *refseq* for NCBI reference sequences, and *obo.so* for Sequence Ontology references.
+#### Important:
+- All new implementations should use validation to ensure consistency with the
+  core specification.
+  When using Schematron, contact *admin* at *varioml.org* for help with
+  generating validation scripts.
+- Use standard ontologies for the content, such as the
+  [Sequence Ontology](http://www.sequenceontology.org/) and
+  [VariO](http://variationontology.org/).
+  Some LSDB specific terms are defined using
+  [SKOS vocabularies](https://github.com/VarioML/VarioML/tree/master/ontology/skos).
+- Source attributes in database xrefs (such as `gene` and `ref_seq`) as well as
+  ontology terms should use database abbreviations defined in the
+  [MIRIAM registry](http://identifiers.org).
+  For example, use `hgnc.symbol` for gene names, `refseq` for NCBI reference
+  sequences, and `obo.so` for Sequence Ontology references.
 
-__VarioML is currently implemented in [Café Variome](http://cafevariome.org).__
-- The specification for the implementation is here: http://varioml.org/cafevariome_minspec.htm.
-- A Schematron implementation is here: https://github.com/VarioML/VarioML/tree/master/xml/cafe_variome/validator
-- An online tool that performs client-side (secure) validation of XML against the Café Variome schema is here: http://varioml.org/validator.htm
 
+#### VarioML (using XML) is implemented in [Café Variome](http://cafevariome.org).
 
-__More information on VarioML can be found at:__ http://varioml.org/
+[//]: # (FIXME: The first and third link here are broken.)
```

Comment:
This should be looked at before this PR can be merged.

Reply:
We should remove the links to varioml.org and use the github directly